CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log - @itwin/access-control-client
 
+## 3.3.1
```

Comment:
This should probably be a minor version change if we follow the [semver](https://semver.org/#summary) standard.